docs/learn/sidebars.ts

```diff
@@ -141,7 +141,7 @@ module.exports = {
       label: 'Governance',
       collapsed: true,
       items: [
-        'governance/shimmer-governance-intro',
+        'governance/governance-intro',
```

Comment:
Let's undo this, it got already fixed in your other PR

```suggestion
        'governance/shimmer-governance-intro',
```